README.md

```diff
@@ -1,18 +1,20 @@
 # colorizer-data
```

Comment:
See here for preview: https://github.com/allen-cell-animated/colorizer-data/blob/doc/getting-started-guide/README.md